modules/administration/pages/subscription-matching.adoc

```diff
@@ -1,153 +1,46 @@
-
-
-
-[[ref.webui.audit.submatch.sub.pinning]]
-== Pins
-
-The subscription pinning feature allows a user to instruct the subscription matcher to favor matching a specific subscription with a given system or group of systems.
-This is achieved by creating pins.
-Each pin contains information about the preferred subscription-system match.
-
-image::audit_subscription_matching_pins.png[scaledwidth=80%]
-
-[NOTE]
-.Respecting Pins
+[IMPORTANT]
 ====
-In some cases the algorithm may determine that a specific pin cannot be respected, depending on the subscription's availability and applicability rules, in this case it will be shown as not satisfied.
+The subscription matcher will not always match clients and subscriptions accurately.
+It is not intended to be a replacement for auditing.
 ====
 
-The pins table displays a list of all pins.
-Items in the list contain the status of pins, which can be ``satisfied``, `not    satisfied` and ``pending next run``.
-
-* A pin is `satisfied` if its system and subscription was matched in the last matcher run.
-* A pin is `not satisfied` if its system and subscription was _not_ matched in the last matcher run.
-This can happen, for example, if the pin violates terms and conditions for subscriptions.
-* A pin is in the `pending next run` state when it needs a new matcher run to be taken into account.
-After the new run, the pin will become either `satisfied` or ``not satisfied``.
-
-image::sm-pins.png[scaledwidth=80%]
-
-Click the btn:[Add a Pin] button to open the [guimenu]``Available Systems`` window.
-You may filter systems by name and select a system for the matcher to pin manually.
-
-image::add-pin.png[scaledwidth=80%]
-
-Within the menu:Subscriptions Available for Selected System[] window click the menu:Save Pin[] button to raise priority for subscription use on the selected system.
-
-
-
-[[ref.webui.audit.submatch.submatch.messages]]
-== Messages
-
-You can review all messages related to [guimenu]``Subscription Matching`` from the menu:Main Menu[Audit > Subscription Matching > Messages] overview.
 
-The following status messages can be displayed.
 
-Unknown Part Number::
-Unsupported part number detected
+== Pin Clients to Subscriptions
 
-Physical Guest::
-Physical system is reported as virtual guest, check hardware data
 
-Guest with Unknown Host::
-Virtual guest has unknown host, assuming it is a physical system
+If the subscription matcher is not automatically matching a particular client with the correct subscription, you can manually pin them.
+When you have created a pin, the subscription matcher favors matching a specific subscription with a given system or group of systems.
 
-Unknown CPU Count::
-System has an unknown number of sockets, assuming 16.
-You can try fixing this by scheduling hardware refresh for affected system.
+However, the matcher will not always respect a pin.
+It depends on the subscription being available, and whether or not the subscription can be applied to the client.
 
```

Comment:
I would add something like this:

A pin will be ignored when it violates the terms and conditions of the matched product, or the algorithm detects a better matching result when ignoring the pin.

Reply:
There is a small typo. Duplicate 'a'

Reply:
Notes on the wording here: I'm trying to avoid 'algorithm' because there's a lot of confusion about what exactly an algorithm is or does (non-technical people seem to consider it as something closer to AI/ML than we do, and I don't think we want to give that impression), so I've stuck with "the matcher" here instead. On "better matching result" I've gone with "a more accurate match", because "better" is really subjective: what makes it better? "More accurate" seems like it's at least a bit more specific than "better". Let me know what you think :)

Reply:
I am fine with 'more accurate'